container-definitions/app.json.tpl

```diff
@@ -2,6 +2,14 @@
   {
     "image": "${image}",
     "name": "${container_name}",
+    %{ if syslog_address != "" }
+    "logConfiguration": {
+      "logDriver": "syslog",
+      "options": {
+        "syslog-address": "${syslog_address}"
+      }
+    },
+    %{else}
```

Comment:
TIL that elseif is [still not a thing](https://github.com/hashicorp/terraform/issues/25313)

Reply:
Yeah I went hunting for it too Lol